docs/reference-guides/core-blocks.md

```diff
@@ -725,7 +725,7 @@ Display a graphic to represent this site. Update the block, and the changes appl
 
 -	**Name:** core/site-logo
 -	**Category:** theme
--	**Supports:** align, color (~~background~~, ~~text~~), ~~alignWide~~, ~~html~~
+-	**Supports:** align, color (~~background~~, ~~text~~), ~~alignWide~~, ~~html~~, spacing (margin, padding)
```

Comment:
This is minor, but can you alphabetize the active supports followed by the alphabetized excluded supports? So spacing would come after color.

Reply:
Nothing too minor. 👍 I addressed this and alpha sorted. Thanks!